mkdocs.yaml

```diff
@@ -25,6 +25,7 @@ theme:
     - navigation.instant.prefetch
     - navigation.instant.preview
     - navigation.instant.progress
+    - navigation.sections
```

Comment:
this makes nav look like:

<img width="594" alt="Screenshot 2024-08-09 at 10 01 10 AM" src="https://github.com/user-attachments/assets/09ff6747-5adc-4bb4-9c7c-8dda55245966">